_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 * 100% of attendees receive the event follow-up email
 
 ## Finding Participants
 
 Your audience for this event should be __people who have written HTML and JavaScript code in the past__.
 
-People with more JavaScript experience are encouraged to help beginners get started. If you have experience developing browser extensions, you may skip the “Build Your First Extension” exercise and focus the activities listed in Part III.
+New to JavaScript and HTML? Check out these tutorials by community members [Viswaprasath Ks](https://www.youtube.com/playlist?list=PLdIyyZgtQZ7CJR5XFB5xvZH0oaBIT_gTD) and [Trishul Goel](https://www.youtube.com/playlist?list=PLI08QU9qtPEJiBs7rVOT4FvXJlisDgTlS) to build your programming skills and create browser extensions! 
+
+
 </div>
 
 <div class="col-md-9" markdown="1">
 
 {: .alert .alert-info .impactbox}
 <span class="glyphicon glyphicon-ok-circle" aria-hidden="true"></span>
-Firefox is an open-source web browser that cares about user control and privacy, and add-ons provide a safe, simple, and powerful way to personalize your browsing experience. From blocking ads to organizing tabs, add-ons help make Firefox your own.
+Firefox is an open-source web browser that cares about user control and privacy, and extensions  provide a safe, simple, and powerful way to personalize your browsing experience. From blocking ads to organizing tabs, add-ons help make Firefox your own.
+
 
-It’s very easy to create add-ons for Firefox using [WebExtensions APIs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions). This Web-based technology framework offers a powerful way to create browser extensions that can be easily ported to Chrome, Opera, and Edge, and you only need to know HTML, JavaScript and CSS to build them.
+It’s very easy to create extensions for Firefox using [WebExtensions APIs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions). This Web-based technology framework offers a powerful way to create browser extensions that can be easily ported to Chrome, Opera, and Edge, and you only need to know HTML, JavaScript and CSS to build them.
 
-Through this activity you’ll help connect new developers with an opportunity to create add-ons to solve real problems and make their lives and the web better.
+In this series of workshops, you’ll help connect new developers with an opportunity to create extensions to solve real problems and make their lives and the web better.
 
 
 ## Planning Your Workshop
 
-Help contribute to the open web by running an extension development workshop. This activity can also be run solo.
+Read the [event guide](https://activate.mozilla.community/eventguide/) on how to set up an event page and how to organize an event. For this campaign, we recommend scheduling a series of 3 workshops and dedicating 1.5 - 4 hours for each workshop. Be sure to complete the actions listed in the Pre-Event section 2 - 3 days before hosting the event. 
```

Comment:
the header will still say "in 1-3 hours". Check the top of this file. I guess it would make sense to update that one as well? Also, would it make sense to change the title to say that it's a series?

Reply:
Hmm. I think it would make sense to change the title to say that it's a series. I think the header can still say it's 1 - 3 hours since that's about how long it will take to complete one event in the series.

Reply:
I guess in that case the text here should be adapted to 1 - 3 hours as well?